src/DIRAC/Core/scripts/dirac_configure.py

```diff
@@ -161,7 +161,13 @@ def setExtensions(self, optionValue):
         return DIRAC.S_OK()
 
     def setIssuer(self, optionValue):
+        # If the user selects the issuer, it means that there will be authorization through tokens
         os.environ["DIRAC_USE_ACCESS_TOKEN"] = "True"
+        # Allow the user to enter only the server domain
+        if not optionValue.startswith("http://") or not optionValue.startswith("https://"):
```

Comment:
```suggestion
        if not optionValue.startswith(("http://", "https://")):
```
https://docs.python.org/3/library/stdtypes.html#str.startswith also takes a tuple